<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rezept</class>
 <widget class="QWidget" name="Rezept">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>531</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1141</width>
    <height>531</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_bereich">
     <property name="text">
      <string>Bereich</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="comboBox_Leistung"/>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="lineEdit_Bemerkung"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_bemerkung">
     <property name="text">
      <string>Bemerkung</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="comboBox_bereich"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_bezeichnung2">
     <property name="text">
      <string>Bezeichnung 2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_Leistung">
     <property name="text">
      <string>Leistung</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="comboBox_bezeichnung2"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Leistung Einf√ºgen</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_position">
     <property name="text">
      <string>Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="comboBox_bezeichnun1"/>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="pushButton_anwenden">
     <property name="text">
      <string>Anwenden</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButton_beenden">
     <property name="text">
      <string>Beenden</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QTextBrowser" name="textBrowser_information"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_bezeichnung1">
     <property name="text">
      <string>Bezeichnung 1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboBox_position"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_information">
     <property name="text">
      <string>Information:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Creator:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="label_3">
     <property name="maximumSize">
      <size>
       <width>110</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../img/dj2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
